torch/export/_trace.py

```diff
@@ -225,6 +247,104 @@ def convert_to_fake(x):
     return fake_args, fake_kwargs, fake_params_buffers, fake_mode
 
 
+def _skip_autograd_kernel(*args, **kwargs):
```

Comment:
this can all be deleted too, right?

Reply:
Hmm this is still needed because this is the one that actually skips autograd.

Reply:
If you want this to be faithful, this needs to be a opoverload.redispatch